inference-engine/src/transformations/include/transformations/op_conversions/convert_mvn1.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
```

Comment:
Why do you include `vector` and `memory` headers in this file?

Reply:
I don't see the resolution for this comment. These includes look redundant.

Reply:
I removed from cpp file and didnt see that the comment was about hpp